jobs/build/gen-assembly/Jenkinsfile

```diff
@@ -85,6 +85,11 @@ node {
                             description: "Take no action, just echo what the job would have done.",
                             defaultValue: false
                         ),
+                        booleanParam(
+                            name: "TRIGGER_BUILD_SYNC",
+                            description: "Automatically trigger build sync against the automatically created PR",
```

Comment:
trigger from pr's branch should use it's ability of DOOZER_DATA_GITREF, If I understand correctly this param indicates to do extra logic(comment) against the branch? I think for build_sync job it should be independent

Reply:
I don't think I get your comment, could you explain better?

This particular PR does not change anything about DOOZER_DATA_GITREF. art-tools `gen-assembly` pipeline will instruct `build-sync` with DOOZER_DATA_GITREF because it needs to run against a non-standard branch.